packages/gasket-react-intl/src/index.d.ts

```diff
@@ -4,49 +4,47 @@ import { LocalePathPartOrThunk, LocaleStatus } from '@gasket/helper-intl';
 export { LocaleStatus };
 
 /**
- * Make an HOC that adds a provider to managing locale files as well as the react-intl Provider.
- * This can be used to wrap a top level React or a Next.js custom App component.
- *
- * @returns wrapper
+ * Make an HOC that adds a provider to managing locale files as well as the
+ * react-intl Provider. This can be used to wrap a top level React or a Next.js
+ * custom App component.
  */
-export function withIntlProvider<Props>(): (Component: React.ComponentType<Props>) => React.ComponentType<Props>;
+export function withIntlProvider<Props>(): (
+  Component: React.ComponentType<Props>
+) => React.ComponentType<Props>;
 
 /**
  * Make an HOC that loads a locale file before rendering wrapped component
- *
- * @param [localePathPart] - Path containing locale files
- * @param [options.loading=null] - Custom component to show while loading
- * @param [options.forwardRef=false] - Forward refs
- * @param [options] - Options
- * @param [options.initialProps=false] - Preload locales during SSR with Next.js pages
- * @returns wrapper
  */
-export function withLocaleRequired<Props>(
+export function withLocaleRequired(
```

Comment:
I believe you need the `Props` generic to inform the `React.ComponentType` return type here.